src/main/java/com/zaxxer/hikari/HikariConfig.java

```diff
@@ -1119,6 +1119,8 @@ else if (prop.matches("scheduledExecutorService|threadFactory") && value == null
             }
             else if (prop.contains("jdbcUrl") && value instanceof String) {
                value = ((String)value).replaceAll("([?&;]password=)[^&#;]*(.*)", "$1<masked>$2");
+               value = ((String)value).replaceAll("([?&;]sslpassword=)[^&#;]*(.*)", "$1<masked>$2");
+               value = ((String)value).replaceAll("([?&;]trustStorePassword=)[^&#;]*(.*)", "$1<masked>$2");
```

Comment:
This change seems reasonable. The url properties are driver dependent so it is conceivable that there are other passwords as well. I wonder if it would be more future-proof to match any property that ends with "assword". What do you think?

Maybe a single replace all with something like this:
```java
               value = ((String)value).replaceAll("([?&;][^&#;=]*assword=)[^&#;]*(.*)", "$1<masked>$2");
```

Reply:
Yeah, almost .. I've updated my PR and added some tests to make sure that also corner cases are covered.